api/lib/source/polkadotV0-source.js

```diff
@@ -807,26 +812,13 @@ class polkadotAPI {
         }
       })
     )
-    const votes = await Promise.all(
```

Comment:
why are `votes` being deleted?

Reply:
they were no votes, just deposits. This fixes the display and the logic in general

Reply:
so the function name is no bueno: `getDemocracyProposalDetailedVotes`